classes/class-wc-connect-settings-pages.php

```diff
@@ -148,8 +155,37 @@ public function output_account_screen() {
 			<?php
 		}
 
+		protected function get_packages_form_data() {
+			return array(
+				'packages' => $this->service_settings_store->get_packages()
+			);
+		}
+
 		public function output_packages_screen() {
+			$store_options = $this->service_settings_store->get_store_options();
+			$root_view = 'wc-connect-packages';
+
+			$admin_array = array(
+				'storeOptions' => $store_options,
+				'formSchema'   => $this->service_schemas_store->get_packages_schema(),
+				'formData'     => $this->get_packages_form_data(),
+				'callbackURL'  => get_rest_url( null, '/wc/v1/connect/packages' ),
+				'nonce'        => wp_create_nonce( 'wp_rest' ),
+				'submitMethod' => 'POST',
+				'rootView'     => $root_view,
+			);
+
+			wp_localize_script( 'wc_connect_admin', 'wcConnectData', $admin_array );
+			wp_enqueue_script( 'wc_connect_admin' );
+			wp_enqueue_style( 'wc_connect_admin' );
+
+			// hiding the save button because the react container has its own
+			global $hide_save_button;
+			$hide_save_button = true;
+
 			?>
+			<div class="wc-connect-admin-container" id="<?php echo esc_attr( $root_view ) ?>">
+			</div>
```

Comment:
To make this consistent with the other React containers, please add the `Having trouble loading this? ...` text.